packages/angular/ssr/test/assets_spec.ts

```diff
@@ -15,10 +15,10 @@ describe('ServerAsset', () => {
   beforeAll(() => {
     assetManager = new ServerAssets({
       bootstrap: undefined as never,
-      assets: {
-        'index.server.html': async () => '<html>Index</html>',
-        'index.other.html': async () => '<html>Other</html>',
-      },
+      assets: new Map([
+        ['index.server.html', async () => '<html>Index</html>'],
+        ['index.other.html', async () => '<html>Other</html>'],
+      ]),
```

Comment:
Consider: `new Map(Object.entries({ /* ... */ }))` can retain the `key: 'value'` literal structure.